lib/jsdom/living/events/PageTransitionEvent.webidl

```diff
@@ -1,6 +1,7 @@
-[Exposed=Window,
- Constructor(DOMString type, optional PageTransitionEventInit eventInitDict)]
+// https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-pagetransitionevent-interface
+[Exposed=Window]
 interface PageTransitionEvent : Event {
+  constructor(DOMString type, optional PageTransitionEventInit eventInitDict = {});
```

Comment:
[ditto](https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-pagetransitionevent-interface)